plugins/onboarding-backend/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/plugin-onboarding-backend",
+  "version": "1.0.0",
```

Comment:
I'd say set this to `0.0.0` as the default, and then a `minor` changeset to take it to `0.1.0`